tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -83,7 +90,7 @@ func (s *stresser) Stress() error {
 					Value: []byte(randStr(s.KeySize)),
 				})
 				putcancel()
-				if grpc.ErrorDesc(err) == context.Canceled.Error() {
+				if checkCancelErr(err) {
```

Comment:
we probably do not want to count failure anymore. previously we only have one long running stresser, so we want to count the overall failure/success. now, when there is a failure we will always return. so it is pointless to do the accounting anymore.